examples/cfd/07_cavity_flow.ipynb

```diff
@@ -0,0 +1,676 @@
+{
```

Comment:
<p>Should we center or...just align all equations?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/devitocodes/devito/pull/1502/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Thanks for the feedback George. Each equation is separated by a new line and seems to be centered. Would aligning the equations mean aligning them at the equals sign? That could be done if its preferred. Thanks!</p>